### PR TITLE
PS-6925: Mismatch in default socket values for mysqld and mysqld_safe

### PR DESCRIPTION
**Pull request by Sudokamikaze**
As *RPM distributions have different path for socket and mysql-client expects it in `/var/lib/mysql/mysql.sock` -- `[mysqld_safe]` should have same one as well.

### CODE REVIEW

**Comment by EvgeniyPatlan** [comment]
As this review requires changes in *.cnf files we need to wait for review from @vadimtk

**Comment by Sudokamikaze** [comment]
Also, there's a different approach if we want to make it have "standard" look: Make a patch which will correct path expectation of socket in mysql-client  and another one for re-locating them to *DEB like locations in .cnf files

**Review by EvgeniyPatlan — APPROVED** [review]
:+1:

**Review by vorsel — APPROVED** [review]

**Comment by EvgeniyPatlan** [comment]
@vadimtk could you please take a look as  it is change in cnf file

**Review by vadimtk — APPROVED** [review]

**Pull request merged**
